src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -33,45 +33,33 @@ export default class AircraftController {
 
     /**
      * @for AircraftController
-     * @method aircraft_generate_callsign
-     * @param airlineName
+     * @method aircraft_auto_toggle
      */
-    aircraft_generate_callsign(airlineName) {
-        // TODO: this should live in the AirlineModel
-        const airline = window.airlineController.airline_get(airlineName);
-
-        if (!airline) {
-            console.warn(`Airline not found: ${airlineName}`);
-
-            return `airline-${airlineName}-not-found`;
-        }
-
-        return airline.generateFlightNumber();
+    aircraft_auto_toggle() {
+        prop.aircraft.auto.enabled = !this.aircraft.auto.enabled;
     }
 
     /**
      * @for AircraftController
-     * @method aircraft_auto_toggle
+     * @method isCallsignInList
+     * @param callsign {string}
+     * return {boolean}
      */
-    aircraft_auto_toggle() {
-        prop.aircraft.auto.enabled = !this.aircraft.auto.enabled;
+    isCallsignInList(callsign) {
+        return this.aircraft.callsigns.indexOf(callsign) !== -1;
```

Comment:
good thought but, unfortunately, `_has` only works with objects.  `indexOf` is [I'm sure] quicker here anyway.  If it 'aint broke don't fix it.

https://lodash.com/docs/4.17.2#has